### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mvwi**
I cleaned up the language to more closely match our other repos but I would love a second (and/or third) set of eyes to make sure that everything is still correct and the examples still work. 🙌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,37 +88,42 @@ public enum NSExceptionCode {
...
+can be updated by running the `gradle pullNetworkConfig` task and committing the updated file.
 
 ## Versioning & release process
 
-##### With the updates to the codebase:  
+### On codebase updates
 
-- [CHANGELOG.md](CHANGELOG.md) should be updated with the listed changes, and 
-a corresponding client version should be bumped in [client.json](src/main/resources/com/unstoppabledomains/client/client.json).
+- [CHANGELOG.md](CHANGELOG.md) will be updated with the listed changes, and
```
Comment (hivtushok) [inline README.md:106]: I think it's better to highlight here that this won't be done automatically, and the corresponding action is required from the contributor (replace 'will' with something like 'required/needed/must/etc').

**Comment by hivtushok** [comment]
@mvwi please, see my comment above, everything else looks fine to me!

File: README.md (hunk 1 above)
Comment (mvwi) [reply README.md]: Updated 👍

**Comment by mvwi** [comment]
Are we ready to merge this?

**Comment by hivtushok** [comment]
> Are we ready to merge this?

Looks good to me 👍🏼

**Pull request merged**